rclcpp/include/rclcpp/time_source.hpp

```diff
@@ -116,89 +116,167 @@ class TimeSource
   RCLCPP_PUBLIC
   void attachClock(rclcpp::Clock::SharedPtr clock);
 
-  /// Detach a clock to the time source
+  /// Detach a clock from the time source
   RCLCPP_PUBLIC
   void detachClock(rclcpp::Clock::SharedPtr clock);
 
+  /// Get whether a separate clock thread is used or not
+  RCLCPP_PUBLIC
+  bool getUseClockThread();
```

Comment:
we use snake case everywhere else for methods, I'm not sure what happened in this class.
I would use snake case in the new methods, even if it's inconsitent with the other methods of the class.

Reply:
Fixed in 8ca01005.

Reply:
thanks!